### ISSUE TITLE
Attention Alignment Not Working

### ISSUE THREAD

**Issue by jinny1208** [Jeevesh8/Cross-Lingual-Voice-Cloning#19]
I am currently training the provided model with Korean and English datasets, with a total of 27 speakers. 
As stated in the README.md, I added Korean to "symbols" as follows:

![image](https://user-images.githubusercontent.com/63538151/159174941-d6fe9260-fb94-462d-b1ff-5e9d236699bc.png)

The problem is that even after training the model for over 45,000 steps, the attention alignment is not forming. 

![image](https://user-images.githubusercontent.com/63538151/159174893-271c89aa-8aaf-40a2-a67a-fdb0bcdd5566.png)

The target and predicted mel-spectrograms seem similar enough.

![image](https://user-images.githubusercontent.com/63538151/159174992-df99e3aa-33cc-4904-8f56-b679c1b7997d.png)

To anyone who has used this repo and to @Jeevesh8 , how long does it normally take for the attention to start aligning properly? Should I continue training?

Any help and advice would be greatly appreciated.

**Comment by Jeevesh8** [comment]
@jinny1208 This is common observed phenomenon[See [here](https://github.com/Rayhane-mamah/Tacotron-2/issues/2340), for example]. The reason is mainly, I think because mel-spectrogram is predicted frame by frame(auto-regressively), so even if the model learns to just copy the last frame of the sequence(without learning anything about alignment), it can easily lower the loss quite a bit. You need to train longer. 

I would suggest you to initialise from Tacotron-2 English pre-trained weights, for faster alignment.

**Comment by jinny1208** [comment]
Hi, thanks for your answer! I will train longer and update the results here

**Comment by jinny1208** [comment]
![image](https://user-images.githubusercontent.com/63538151/160747004-ad8fc9ee-a6c1-4ad3-b35a-28de36691e05.png)

The alignment works to some degree. I probably need to train longer and do somthing else to get clearer and more robust alignments.
